lib/fastlane/plugin/menigit/actions/add_git_remote.rb

```diff
@@ -0,0 +1,62 @@
+require 'fastlane'
+
+module Fastlane
+  module Actions
+    class AddGitRemoteAction < Action
+      def self.run(options)
+        remote = options[:remote]
+        path = options[:path]
+        UI.user_error!("No remote name given, pass using `remote: 'name'`") unless remote.to_s.length > 0
+        UI.user_error!("No path given, pass using `path: 'path'`") unless path.to_s.length > 0
+
+        UI.message("Adding remote #{remote} '#{path}'' 🎯.")
```

Comment:
Are those `' ''` needed or in proper numbers?

Reply:
'`' removed